testsuite/MDAnalysisTests/topology/test_top.py

```diff
@@ -85,6 +88,23 @@ def test_impropers_atom_counts(self, filename):
         assert len(u.atoms[[self.atom_i]].impropers) == \
             self.expected_n_i_impropers
 
+    def test_chainIDs(self, filename):
```

Comment:
rather than have this in a parent class that will be only used by one child class, could we just put this in the child class only?